docs/jwst/error_propagation/main.rst

```diff
@@ -0,0 +1,374 @@
+optional arrays of wavelength and relative response (as a function of wavelength).
+The combination of the scalar conversion factor and the 2-D response values is
+applied to the science data, including the SCI and ERR arrays, as well as the
+variance (VAR_POISSON, VAR_RNOISE, and VAR_FLAT) arrays. The correction values
+are divided into the SCI and ERR arrays, and the square of the correction values 
+are divided into the variance arrays.  For details of the photom correction, see 
+`Photom
+<https://jwst-pipeline.readthedocs.io/en/latest/jwst/photom/index.html>`_
+
+
+Stage 3 pipelines
+-----------------
+Stage 3 pipelines perform corrections that work with multiple exposures and in
+most cases produce some kind of combined product).  There are unique pipelines
+for imaging, spectroscopic, coronographic, AMI and TSO observations. OMIT ??>> For details
+of these pipelines see <add link here>.
+
+
+Stage 3 Image Processing Pipeline - calwebb_image3
+++++++++++++++++++++++++++++++++++++++++++++++++++
+This pipeline is for non-TSO imaging only, and combines the calibrated data 
+from multiple exposures (e.g. a dither or mosaic pattern) into a single rectified
+(distortion corrected) product.  The steps in this pipeline are tweakreg, 
+skymatch, outlier_detection, resample, source_catalog.  The only one of these 
+steps that either uses or modifies variance/error arrays that are propagated
+through the pipeline is outlier_detection, described here:
+
+Outlier_detection 
+~~~~~~~~~~~~~~~~~
+This step uses the ERR array to make a local noise model, based on the readnoise
+and calibration errors of earlier steps in the pipeline. This step does not modify
+the ERR array or any of the VAR arrays.
+
+
+Stage 3 Spectroscopic Processing Pipeline - calwebb_spec3
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+This pipeline is for non-TSO imaging only, and is intended for combining the
+calibrated data from multiple exposures (e.g. a dither/nod pattern) into a
+single combined 2D or 3D spectral product and a combined 1D spectrum.  The steps 
+in this pipeline are master_background, exp_to_source, mrs_match, 
+outlier_detection, resample_spec, cube_build, extract_1d, and combine_1d.  The
+paricular steps executed depend on particular instruments or instrument modes; 
+for details see `Calwebb_spec3
+<https://jwst-pipeline.readthedocs.io/en/latest/jwst/pipeline/calwebb_spec3.html>`_
+
+Of the steps listed above, below are the steps that either use or modify
+variance/error arrays that are propagated through these pipelines.
+
+Outlier_detection 
+~~~~~~~~~~~~~~~~~
+This step uses the ERR array to make a local noise model, based on the readnoise
+and calibration errors of earlier steps in the pipeline. This step does not modify
+the ERR array or any of the VAR arrays.
+
+Cube_build
+~~~~~~~~~~
+Cube_build takes MIRI or NIRSpec IFU calibrated 2-D images and produces 3-D
```

Comment:
Add a short into saying "The cube_build step is only applied in the Stage 3 Spectroscopic (calwebb_spec3) pipeline. It takes MIRI or ..."